src/inngest/functions/transcript-ready.ts

```diff
@@ -52,14 +54,41 @@ export const transcriptReady = inngest.createFunction(
           .commit()
       })
 
-      await step.sendEvent('announce that srt is ready', {
-        name: MUX_SRT_READY_EVENT,
-        data: {
-          videoResourceId: videoResource._id as string,
-          moduleSlug: event.data.moduleSlug,
-          srt: event.data.srt,
+      let eggAxios = axios.create({
+        baseURL: EGGHEAD_AUTH_DOMAIN,
+        headers: {
+          Authorization: `Bearer ${railsToken}`,
         },
       })
+
+      if (event.data.eggheadLessonId) {
+        await step.run(
+          'add transcript and srt to lesson in egghead',
+          async () => {
+            let lessonParams = {
+              'lesson[srt]': event.data.srt ?? '',
+              'lesson[transcript]': event.data.transcript ?? '',
+            }
+
+            let body = new URLSearchParams(lessonParams)
+
+            return await eggAxios.put(
+              `/api/v1/lessons/${event.data.eggheadLessonId}`,
+              body,
+            )
+          },
+        )
+      }
```

Comment:
if `eggheadLessonId` is present then we want to post srt and transcript to that lesson object